### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by dav1312**
- [x] Dark theme buttons
- [x] Dark theme diff
- [x] Softer elo results colors on dark theme
- [x] Change buttons order
- [x] Removed some inline styles
- [x] Use variables for common values in dark theme
- [x] Changed pagination style
- [x] Custom Scrollbars
- [x] Navbar icons (from Fontawesome)
- [x] Scrollable navbar (not fixed to the rest of the website)
- [x] Changed links color (to green)
- [x] Moved tables inside divs for overflow management
- [x] Sticky table headers
- [x] Bigger elo-results inside test view
- [x] Aligned run date and diff
- [x] Other small fixes

### CODE REVIEW

**Comment by ppigazzini** [comment]
Weird, I had not conflict merging on DEV...

**Comment by ppigazzini** [comment]
It's fun with my free spin wheel logi mouse!

**Comment by ppigazzini** [comment]
@linrock @vondele @snicolet @SFisGOD please have a look at your convenient time :)

**Comment by tomtor** [comment]
Not related to this PR but the `Eval Guide` link https://hxim.github.io/Stockfish-Evaluation-Guide/ is broken?

**Comment by ppigazzini** [comment]
> Not related to this PR but the `Eval Guide` link https://hxim.github.io/Stockfish-Evaluation-Guide/ is broken?

It works for me on DEV and PROD

**Comment by ppigazzini** [comment]
On mobile portrait mode some pages (eg single test view top half page) need a horizontal scroll to not crop the content.

In PC mode the left menu perhaps could be narrowed a bit to gain space.

**Comment by ppigazzini** [comment]
> When using a small window the icons on the left miss some white space to the right:

What about dropping that scroll bar as in the mobile theme?

**Comment by vdbergh** [comment]
For me the experience on my iPad is not so good (light mode).

The sidebar is too wide (double as wide as necessary).

The main table is too wide, making horizontal scrolling necessary (it seems the tables do not adapt to the width of the screen).

The last item is a question of taste. The divs scroll vertically behind each other. Since they have no border I find it a bit confusing to see what goes on.

I am including a screenshot below.

File_000

File: server/fishtest/static/css/application.css [hunk 1]
```diff
@@ -43,43 +134,246 @@ legend {
...
+
+.mainnavbar [href="/tests/finished?ltc_only=1"]::before {
+  content: "\f017";
+}
+
+.mainnavbar [href="/tests/finished?ltc_only=1&success_only=1"]::before {
+  content: "\f06d";
+}
+
+.mainnavbar [href="/tests/user/MFpOk27y"]::before {
```
[the remark this answers is not in the document]
Comment (dav1312) [reply server/fishtest/static/css/application.css:244]: It is and isn't at the same time

**Comment by ppigazzini** [comment]
Applied to PROD by hand to collect opinions, suggestions, requests etc.

**Comment by Sopel97** [comment]
Some feedback and suggestions. Firefox, fullscreen, full hd.

- Double scrollbar in network list seems redundant, and with the current size it makes it hard to scroll to the bottom
obraz
- Changing ordering only changes the current page, disregards older nets.
- Overview has nested scrollbars, it's very hard to navigate with mouse wheel.
obraz
- Same goes for the game batch list in a test page
- Consider putting the machine list to a different page
- rename "NN Stats" to "Networks"
- The https://dfts-0.pigazzini.it/html/live_elo.html?616eab99942d40685e323779 page could be merged with https://dfts-0.pigazzini.it/tests/view/616eab99942d40685e323779?
- Some pages have useless double scrollbar, one inactive.

**Comment by ppigazzini** [comment]
@Sopel97 thank you for the feedbacks!
We have these links: "Upload NN", "NN Trainer", what about renaming "NN Stats" ->"NN Repo" ? Internally the page has been already renamed "Neural Network Repository"

**Comment by vondele** [comment]
yes, also good IMO.

for the test page, and again not necessarily part of this PR, I would prefer to have the diff displayed above the details section, i.e. next to the approve button. Now, I always need to scroll down before approving, for me the diff is the essential bit to approve.

**Pull request merged**